src/core/ext/xds/xds_api.cc

```diff
@@ -99,6 +99,17 @@ bool XdsTimeoutEnabled() {
   return parse_succeeded && parsed_value;
 }
 
+// TODO(yashykt): Check to see if xDS security is enabled. This will be
+// removed once this feature is fully integration-tested and enabled by
+// default.
+bool XdsSecurityEnabled() {
+  char* value = gpr_getenv("GRPC_XDS_EXPERIMENTAL_SECURITY_SUPPORT");
```

Comment:
Please make sure this is the same env var name being used in other languages.

Reply:
Yes, this is what Java is using. Go has not introduced this yet.